tests/test_routes.py

```diff
@@ -10,6 +10,73 @@
 #  UTF-8
 # ------------------------------------------------------------ #
 
+def test_shorthand_routes():
+    app = Sanic('test_shorhand_routes')
+
+    @app.get('')
```

Comment:
Could we split these into separate tests? Perhaps parameterize them so that we just write one test to do this?